frontend/components/pod_components/TraineePod.js

```diff
@@ -2,75 +2,121 @@ import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, Text, View, TouchableOpacity, SafeAreaView } from 'react-native';
 import Constants from 'expo-constants';
-import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
+import axios from 'axios';
 
-export default function TraineePodScreen({ navigation }) {
-    return (
-        <SafeAreaView style={styles.container}>
-            <Text style={styles.title}>
-                TRAINEE POD
-            </Text>
-            
-            <TouchableOpacity 
-                style={styles.block} 
-                onPress={() => navigation.navigate('Random Screen')}
-            >
-                <Text style={styles.blockText}>
-                    Competencies
-                </Text>
+import PodProgressBar from './PodProgressBar.js';
+
+const COMPET_TOTAL_OUTCOMES = 3
+const CAREER_TOTAL_OUTCOMES = 2
+const LIFE_TOTAL_OUTCOMES = 2
```

Comment:
This works for now, but when we move away from hard coding this you will need to think about potentially writing an endpoint that could give you the total amount for each outcome.